omod-common/src/main/java/org/openmrs/module/webservices/rest/util/ReflectionUtil.java

```diff
@@ -17,7 +17,13 @@
 import java.lang.reflect.ParameterizedType;
 import java.lang.reflect.Type;
 import java.lang.reflect.TypeVariable;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 
+import com.sun.tools.javac.util.Pair;

```

Comment:
Is there any Pair-like class that I can use here? I had previously added a dependency to javatuples but was told to remove that as well. I know it's not a big deal and I can just concat the two into a String or something, but this will be the second time that a fix has done this in the restws module; at some point doesn't it just make sense to add a dependency which provides this?